deluge/core/core.py

```diff
@@ -770,9 +776,9 @@ def get_torrent_status(
             all_keys=not keys,
         )
 
-    @export
+    @export  # type: ignore
```

Comment:
Why are we ignoring type here and below?

Reply:
I did a test merging my mypy stuff with the maybe_coroutine branch, and type hinting is muuuch nicer (than inlineCallbacks.) No need to wrap stuff in `Deferred[returntypehere]` no need to document generators with weird generator type hints where you have to specify the yield, send, and return types, all just works.